.github/workflows/python-tox.yml

```diff
@@ -24,57 +23,15 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        tox_env:
-          # Test 2.2 (LTS), 3.1 (Latest) releases for Python 3.6, 3.7
-          - py36-django22-wagtail27
-          - py36-django31-wagtail27
-          - py36-django22-wagtail210
-          - py36-django31-wagtail210
-          - py37-django22-wagtail27
-          - py37-django31-wagtail27
-          - py37-django22-wagtail210
-          - py37-django31-wagtail210
-
-          # Test all versions for Python 3.8
-          - py38-django22-wagtail27
-          - py38-django30-wagtail27
-          - py38-django31-wagtail27
-          - py38-django22-wagtail210
-          - py38-django30-wagtail210
-          - py38-django31-wagtail210
-
-
-        include:
-          - python-version: "3.6"
-            tox_env: py36-django22-wagtail27
-          - python-version: "3.6"
-            tox_env: py36-django31-wagtail27
-          - python-version: "3.6"
-            tox_env: py36-django22-wagtail210
-          - python-version: "3.6"
-            tox_env: py36-django31-wagtail210
-
-          - python-version: "3.7"
-            tox_env: py37-django22-wagtail27
-          - python-version: "3.7"
-            tox_env: py37-django31-wagtail27
-          - python-version: "3.7"
-            tox_env: py37-django22-wagtail210
-          - python-version: "3.7"
-            tox_env: py37-django31-wagtail210
-
-          - python-version: "3.8"
-            tox_env: py38-django22-wagtail27
-          - python-version: "3.8"
-            tox_env: py38-django30-wagtail27
-          - python-version: "3.8"
-            tox_env: py38-django31-wagtail27
-          - python-version: "3.8"
-            tox_env: py38-django22-wagtail210
-          - python-version: "3.8"
-            tox_env: py38-django30-wagtail210
-          - python-version: "3.8"
-            tox_env: py38-django31-wagtail210
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Why are Python versions separated by points, but not Django or Wagtail versions?

Reply:
Good question! Looking at the code I see no technical reasons. It's most likely me being effected by the Github CI sample: https://docs.github.com/en/actions/guides/building-and-testing-python

There should be one way of doing this, so I'll look into converting them back to integers, makes more sense.

Reply:
I finally got the time to reactor the python definition so the python version in `tox.ini` are defined without the dot. I did have to keep a dot separated python version around in github actions so that `actions/setup-python@v1`would pick the proper python version, but that change is localized in the CI config and not tox.